beacon-chain/forkchoice/doubly-linked-tree/reorg_late_blocks_test.go

```diff
@@ -0,0 +1,79 @@
+package doublylinkedtree
+
+import (
+	"context"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v3/config/params"
+	"github.com/prysmaticlabs/prysm/v3/testing/require"
+)
+
+func TestForkChoice_ShouldOverrideFCU(t *testing.T) {
```

Comment:
We are missing an important test case where head arrives early

Reply:
```go
	if early {
		return
	}
```

Reply:
nice!